crates/cudart-sys/src/utils.rs

```diff
@@ -4,12 +4,15 @@ use std::path::{Path, PathBuf};
 pub fn get_cuda_path() -> Option<&'static Path> {
     #[cfg(target_os = "linux")]
     {
-        let path = Path::new("/usr/local/cuda");
-        if path.exists() {
-            Some(path)
-        } else {
-            None
+        for path_name in [option_env!("CUDA_PATH"), Some("/usr/local/cuda")].iter().flatten() {
+            println!("trying {path_name}...");
```

Comment:
This function is used by other functions below and is getting called multiple times during a build process.
IMHO it should not be that verbose, otherwise we end up with a multiline prints repeated several times. My preference would be to not print anything.